.github/workflows/build.yaml

```diff
@@ -21,9 +21,23 @@ jobs:
       - name: Run formater and linter
         run: npm run lint
 
-  build:
+  test:
     runs-on: ubuntu-latest
     needs: lint
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: "npm"
+      - run: npm ci             # install dependencies
+      - run: npm run test:unit
```

Comment:
We should change this to just `test`. This would comply with our [refarch template](https://github.com/it-at-m/refarch-templates/blob/b909e68f467542f7e4679fab3a08baa883362b06/refarch-frontend/package.json#L11)

Reply:
Done with [ed7b5c5](https://github.com/it-at-m/muc-patternlab-vue/pull/315/commits/ed7b5c519b21ed90215cd7024cd891802c92deaa)